projects/core/components/tooltip/tooltip.template.html

```diff
@@ -1,11 +1,16 @@
-<tui-svg
-    src="tuiIconTooltipLarge"
+<span
```

Comment:
@waterplea what do you think, `span` is better than `div` in terms of html semantics?

Reply:
Yes, like I said in another PR, if we use `div` here just putting `tui-tooltip` in a `p` tag will make it invalid:
![image](https://user-images.githubusercontent.com/11832552/150935980-9c24c1fc-8ce8-449c-bec0-c3f0dce8d987.png)

Not that we follow it everywhere but still it's better to keep that in mind.

Reply:
sure, thank you for explain ))